frontend/src/components/uam/components/users-table/users-table.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement } from 'react';
+import { Column } from 'react-table';
+import { Table } from '../../../common/common';
+import styles from './styles.module.scss';
+
+type Props<Data extends Record<string, unknown>> = {
+  columns: Column<Data>[];
+  data: readonly Data[];
+};
+
+const UsersTable = <Data extends Record<string, unknown>>({
+  columns,
+  data,
+}: Props<Data>): ReactElement => {
```

Comment:
If it's users table, then you already know type of displayed data, it should be generic
```suggestion
type Props = {
  columns: Column<{existing_user_type}>[];
  data: readonly {existing_user_type}[];
};

const UsersTable = ({
  columns,
  data,
}: Props): ReactElement => {
```